incuna-sass/helpers/_link.sass

```diff
@@ -1,16 +1,16 @@
 // Mixin to quickly add / remove default styles to links
 @mixin link($type: default)
     cursor: pointer
-    @if $type = default
```

Comment:
The issue here is not a missing `@else`, it is because you are assigning the `$type` variable to a value, not comparing it. An assignment will always return true, so all conditions are true and all `@if` statements ran, with the last one taking precedence due to the C(ascade)SS. `@else` won't run because `@if` is true.